presto-bigquery/pom.xml

```diff
@@ -358,6 +376,9 @@
                                 <exclude>com.fasterxml.jackson.core:jackson-core</exclude>
                                 <exclude>javax.annotation:javax.annotation-api</exclude>
                                 <exclude>com.fasterxml.jackson.core:jackson-databind</exclude>
+                                <exclude>com.google.auth:google-auth-library-oauth2-http</exclude>
+                                <exclude>com.google.auth:google-auth-library-credentials</exclude>
+                                <exclude>org.conscrypt:conscrypt-openjdk-uber</exclude>
```

Comment:
@imjalpreet  Added those exclusion to resolve Require upper bound dependencies error in bigquery pom.xml . Those were included in Presto as part of other dependency.

Reply:
Can you please share the error we saw when we didn't exclude it? I am curious since I can see the same dependency in both the dependency trees.

Reply:
[logs.docx](https://github.com/user-attachments/files/17675502/logs.docx)

@imjalpreet  Please see the above log file.

Reply:
@nishithakbhaskaran Thank you for sharing the logs. Can we assess how complex it would be to fix these errors instead of simply excluding them from the check?

Let's try upgrading all three components to their latest required versions based on the error log and see if any new issues arise.

Upgrade `com.google.auth:google-auth-library-oauth2-http` to `1.22.0` 

Add `com.google.auth:google-auth-library-credentials`  to dependencyManagement with version `1.22.0` 

Add `org.conscrypt:conscrypt-openjdk-uber` to dependencyManagement with version `2.5.2`

Reply:
@imjalpreet  I have updated the same as suggested, but after that it shows lot more dependency with Require upper bound dependencies error in the bigquery pom.

Reply:
Hi @imjalpreet @tdcmeehan  Addressed the comments. Along with that I need to handle some other upperbound dependency failures in the dependencymanagement tag.